### ISSUE TITLE
Update stored count when emptying empty pouches via chat message 

### ISSUE THREAD

**Issue by Infinitay** [Infinitay/essence-pouch-degrading#14]
Related to #13 

Whenever I was facing the bug where the count wasn't updating when I was quickly emptying and RCing I noticed that when emptying an empty essence pouch it wasn't updating the count to 0 because the current implementation does it only when we check it specifically.

**Comment by Infinitay** [comment]
Further thoughts is that this was due to #13 so it didn't realized I emptied it. In other words, the pouch itself was emptied of essence, but the state wasn't properly updated so it thought there was essence still in it and our inventory state also didn't change. Therefore it wasn't emptying the essence and had the wrong counts on it.

tldr just make it so clicking empty would also trigger a check for chat message in instances where someone isn't checking and just empties their pouch with an unknown or stale state
